applications/matter_weather_station/README.rst

```diff
@@ -225,7 +226,7 @@ To build this application to work with the nRF7002 Wi-Fi expansion board:
 Generating factory data
 =======================
 
-To enable factory data support, you need to select the ``factory_data`` build type from the available application :ref:`build types <matter_weather_station_app_build_types>`.
+To enable factory data support, you need to select the ``overlay-factory_data`` overlay from the available application :ref:`build configuration overlays <matter_weather_station_build_configuration_overlays>`.
```

Comment:
```suggestion
To enable factory data support, you need to select the ``overlay-factory_data`` configuration overlay from the available application :ref:`build configuration overlays <matter_weather_station_build_configuration_overlays>`.
```
You need to also add the target tag for the new heading. The missing tag is what is currently breaking the doc build.

Reply:
Right, thanks for the catch.